testing/zerotier-one/zerotier-one.pre-install

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+/usr/sbin/addgroup -S zerotier-one
+/usr/sbin/adduser -D -H -S -h /var/lib/zerotier-one -s /sbin/nologin -g 'https://www.zerotier.com/' -G zerotier-one zerotier-one
```

Comment:
This actually follows common pattern we currently use in most of the packages. Except this GECOS with URL, this should be removed.